config/locales/views/imports/uk.yml

```
@@ -0,0 +1,468 @@
+      csv_uploaded: "CSV успішно завантажено."
+      ndjson_uploaded: "Файл NDJSON успішно завантажено."
+      pdf_too_large: "PDF-файл завеликий. Максимальний розмір — %{max_size}MB."
+      pdf_processing: "Ваш PDF обробляється. Ви отримаєте електронний лист, коли аналіз буде завершено."
+      invalid_pdf: "Завантажений файл не є дійсним PDF."
+      duplicate_pdf_unavailable: "Цей PDF уже збережено як виписку, до якої ви не маєте доступу."
+      document_too_large: "Файл документа завеликий. Максимальний розмір — %{max_size}MB."
+      invalid_document_file_type: "Недійсний тип файлу документа для активного векторного сховища."
+      document_uploaded: "Документ успішно завантажено."
+      document_upload_failed: "Не вдалося завантажити документ до векторного сховища. Будь ласка, спробуйте ще раз."
+      invalid_ndjson_file_type: "Недійсний тип файлу або формат. Будь ласка, завантажте дійсний файл експорту .ndjson або .json."
+      ndjson_uploaded: "Файл NDJSON успішно завантажено."
```

YAMLlint (1.37.1)

[error] duplication of key "ndjson_uploaded" in mapping

(key-duplicates)